rasa/core/policies/rule_policy.py

```diff
@@ -34,6 +34,11 @@
     USER_INTENT_SESSION_START: ACTION_SESSION_START_NAME,
 }
 
+RULES = "rules"
+NEGATIVE_RULES = "negative_rules"
+NO_VALIDATION = "no_validation"
```

Comment:
to make it clearer:
```suggestion
FORM_EXECUTION_AFTER_REJECTION = "no_validation"
```

Reply:
hm, I don't see how `FORM_EXECUTION_AFTER_REJECTION` is related to no validation

Reply:
renamed to `DO_NOT_VALIDATE_FORM`